src/main/java/bot/Ui.java

```diff
@@ -0,0 +1,318 @@
+package bot;
+
+import java.util.ArrayList;
+
+/**
+ * Class that represents the UI of 4LC3N-BOT
+ */
+public class Ui {
+    public static final String VERSION = "4LC3N v0.1.0";
+    private static final String LOAD =
+            "=======================================================\n"
```

Comment:
You could further declare a static final constant for the line feed character `\n`, as line  feeds defer on different systems too. That would enable greater portability.